platforms/win32 broken on non-Windows platforms

```
> python vivisect/vivbin.py --bulk samples/004e32a0f3610532cf08ce0ca4ee3ce2628faa0a143b732151706a8b7fda0df1                                                                            
Traceback (most recent call last):
  File "vivisect/vivbin.py", line 144, in <module>
    main()
  File "vivisect/vivbin.py", line 98, in main
    vw.loadFromFile(fname, fmtname=args.parsemod)
  File "/home/user/code/vivisect/vivisect/__init__.py", line 2724, in loadFromFile
    mod = viv_parsers.getParserModule(fmtname)
  File "/home/user/code/vivisect/vivisect/parsers/__init__.py", line 89, in getParserModule
    __import__(mname)
  File "/home/user/code/vivisect/vivisect/parsers/pe.py", line 13, in <module>
    import vtrace.platforms.win32 as vt_win32
  File "/home/user/code/vivisect/vtrace/platforms/win32.py", line 32, in <module>
    from ctypes import wintypes
  File "/usr/lib/python3.8/ctypes/wintypes.py", line 20, in <module>
    class VARIANT_BOOL(ctypes._SimpleCData):
ValueError: _type_ 'v' not supported
```

see: https://stackoverflow.com/a/63274446/87207 and https://bugs.python.org/issue16396

ultimately, as far as i can tell, `ctypes.wintypes` should not be used on anything but Windows. so, its not appropriate to be used in [vtrace.platform.win32](https://github.com/vivisect/vivisect/blob/726afe014aee77dafe745380daa293c363b3eae3/vtrace/platforms/win32.py#L32). we may need to revert a portion of #404 that introduced these changes. also, I'd recommend configuring CI to also run on Linux/MacOS so similar issues can be caught sooner.

PR #416 shows the same test behavior as master, no regressions.

```
(env) PS C:\Users\user\Downloads\vivisect> git log -1
commit a9230c69199f7f5d0c3946480cdc0da3614734f7 (HEAD -> fix-415)
Author: William Ballenthin <william.ballenthin@fireeye.com>
Date:   Mon Aug 9 19:33:06 2021 -0600

    win32: remove wintypes import

    closes #415
(env) PS C:\Users\user\Downloads\vivisect> python -V
Python 3.8.5

======================================================================
ERROR: test_cobra_proxy (cobra.tests.testbasic.CobraBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\cobra\tests\testbasic.py", line 21, in test_cobra_proxy
    daemon = cobra.CobraDaemon(port=60600)
  File "C:\Users\user\Downloads\vivisect\cobra\__init__.py", line 512, in __init__
    ThreadingTCPServer.__init__(self, (host, port), CobraRequestHandler)
  File "c:\users\user\appdata\local\programs\python\python38\lib\socketserver.py", line 452, in __init__
    self.server_bind()
  File "c:\users\user\appdata\local\programs\python\python38\lib\socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions
    self._target(*self._args, **self._kwargs)

======================================================================
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
ERROR: test_cobra_newobj (cobra.tests.testnewobj.CobraNewObjTest)
----------------------------------------------------------------------
    self.platformProcessEvent(event)
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\cobra\tests\testnewobj.py", line 18, in test_cobra_newobj
    daemon = cobra.CobraDaemon(port=60500, msgpack=True)
  File "C:\Users\user\Downloads\vivisect\cobra\__init__.py", line 512, in __init__
    ThreadingTCPServer.__init__(self, (host, port), CobraRequestHandler)
  File "c:\users\user\appdata\local\programs\python\python38\lib\socketserver.py", line 452, in __init__
    self.server_bind()
  File "c:\users\user\appdata\local\programs\python\python38\lib\socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent

======================================================================
ERROR: test_emu_lockstep (envi.tests.test_emu_lockstep.LockStepTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

    self.platformDetach()
======================================================================
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
ERROR: test_vdb_stalker (vdb.tests.teststalker.VdbStalkerTest)
----------------------------------------------------------------------
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vdb\tests\teststalker.py", line 19, in test_vdb_stalker
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 62, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach

======================================================================
ERROR: test_symboliks_reduce_op_and (vivisect.symboliks.tests.test_reduce.TestReduceCase)
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\tests\test_reduce.py", line 34, in test_symboliks_reduce_op_and
    self.assertReduce('foo[5] & 0', '0')
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\tests\test_reduce.py", line 11, in assertReduce
    sym1 = symexp(s1).reduce(foo=True)
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\expression.py", line 40, in symexp
    return p.parseExpression(expr)
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\expression.py", line 67, in parseExpression
    return self.astToSymboliks(a.body[0])
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\expression.py", line 90, in astToSymboliks
    return self.astToSymboliks(a.value)
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\expression.py", line 93, in astToSymboliks
    left = self.astToSymboliks(a.left)
  File "C:\Users\user\Downloads\vivisect\vivisect\symboliks\expression.py", line 135, in astToSymboliks
    raise Exception('Unsupported Expression (symbolik width index)')
Exception: Unsupported Expression (symbolik width index)

======================================================================
ERROR: test_mimikatz_segments (vivisect.tests.testpe.PETests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vivisect\tests\testpe.py", line 501, in test_mimikatz_segments
    self.assertEqual(ans[sname][0], sva)
KeyError: '.rsrc'

======================================================================
ERROR: test_firefox_segments (vivisect.tests.testvivisect.VivisectTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vivisect\tests\testvivisect.py", line 830, in test_firefox_segments
    self.assertEqual(ans[sname][0], sva)
KeyError: '.rsrc'

======================================================================
ERROR: vqt.tests.testmenubuilder (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: vqt.tests.testmenubuilder
Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python38\lib\unittest\loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "c:\users\user\appdata\local\programs\python\python38\lib\unittest\loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "C:\Users\user\Downloads\vivisect\vqt\tests\testmenubuilder.py", line 3, in <module>
    import vqt.menubuilder as vmb
  File "C:\Users\user\Downloads\vivisect\vqt\menubuilder.py", line 3, in <module>
    from PyQt5.QtWidgets import *
ModuleNotFoundError: No module named 'PyQt5'


======================================================================
ERROR: vstruct.qt (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: vstruct.qt
Traceback (most recent call last):
  File "c:\users\user\appdata\local\programs\python\python38\lib\unittest\loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "c:\users\user\appdata\local\programs\python\python38\lib\unittest\loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "C:\Users\user\Downloads\vivisect\vstruct\qt\__init__.py", line 4, in <module>
    import vqt.tree as vq_tree
  File "C:\Users\user\Downloads\vivisect\vqt\tree.py", line 1, in <module>
    from PyQt5 import QtCore
ModuleNotFoundError: No module named 'PyQt5'


======================================================================
ERROR: test_baselib (vtrace.tests.test_expressions.VtraceExpressionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_sym (vtrace.tests.test_expressions.VtraceExpressionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_breakpoint (vtrace.tests.testbasic.VtraceBasicExecTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 38, in test_vtrace_breakpoint
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 90, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_exename (vtrace.tests.testbasic.VtraceBasicExecTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 83, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_getregisters (vtrace.tests.testbasic.VtraceBasicExecTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 18, in test_vtrace_getregisters
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 90, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_mmaps (vtrace.tests.testbasic.VtraceBasicExecTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 83, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_setregisters (vtrace.tests.testbasic.VtraceBasicExecTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 27, in test_vtrace_setregisters
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 90, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_breakpoint (vtrace.tests.testbasic.VtraceBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 38, in test_vtrace_breakpoint
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 62, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_exename (vtrace.tests.testbasic.VtraceBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_getregisters (vtrace.tests.testbasic.VtraceBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 18, in test_vtrace_getregisters
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 62, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_mmaps (vtrace.tests.testbasic.VtraceBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_setregisters (vtrace.tests.testbasic.VtraceBasicTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testbasic.py", line 27, in test_vtrace_setregisters
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 62, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_emulatorFromTrace (vtrace.tests.tests_envitools.EnvitoolsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 31, in tearDown
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_threads (vtrace.tests.testthread.VtraceThreadTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testthread.py", line 30, in test_vtrace_threads
    self.runUntilExit()
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\__init__.py", line 62, in runUntilExit
    self.trace.run()
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 283, in run
    self.wait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 167, in wait
    self._doWait()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 177, in _doWait
    self.platformProcessEvent(event)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1684, in platformProcessEvent
    self.platformDetach()
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 890, in trfunc
    raise ret
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\base.py", line 924, in run
    queue.put(meth(*args, **kwargs))
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1440, in platformDetach
    kernel32.CloseHandle(phandle)
ctypes.ArgumentError: argument 1: <class 'TypeError'>: wrong type

======================================================================
ERROR: test_vtrace_writemem (vtrace.tests.testwritemem.VtraceWritememTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vtrace\tests\testwritemem.py", line 22, in test_vtrace_writemem
    self.trace.writeMemory(addr, testbuf)
  File "C:\Users\user\Downloads\vivisect\vtrace\__init__.py", line 526, in writeMemory
    self.platformWriteMemory(int(address), bytez)
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1528, in platformWriteMemory
    raiseWin32Error("kernel32.WriteProcessMemory")
  File "C:\Users\user\Downloads\vivisect\vtrace\platforms\win32.py", line 1205, in raiseWin32Error
    raise vtrace.PlatformException("Win32 Error %s failed: %s" % (name,kernel32.GetLastError()))
vtrace.PlatformException: Win32 Error kernel32.WriteProcessMemory failed: 998

======================================================================
FAIL: test_memorymaps_basicperms (vivisect.tests.testmalware.MalwareTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vivisect\tests\testmalware.py", line 36, in test_memorymaps_basicperms
    self.assertEqual(maps, set(vw.getMemoryMaps()))
AssertionError: Items in the second set but not the first:
(269000704, 436, 4, '0e25aa791c9119108af073bc9e9d0fa2')

======================================================================
FAIL: test_basic_apis (vivisect.tests.testvivisect.VivisectTest)
Test a bunch of the simpler workspace APIs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\user\Downloads\vivisect\vivisect\tests\testvivisect.py", line 240, in test_basic_apis
    self.assertEqual(locdist, ans[loctype])
AssertionError: Tuples differ: ('Undefined', 0, 259593, 44) != ('Undefined', 0, 53337, 14)

First differing element 2:
259593
53337

- ('Undefined', 0, 259593, 44)
?                  - ---   ^

+ ('Undefined', 0, 53337, 14)
?                    +++  ^


----------------------------------------------------------------------
Ran 353 tests in 364.471s

FAILED (failures=2, errors=24, skipped=2)
```

--- a/vtrace/platforms/win32.py
+++ b/vtrace/platforms/win32.py
@@ -56,6 +56,7 @@ BOOL    = c_ulong
 BYTE    = c_ubyte
 NULL    = 0
 LPCWSTR = LPWSTR = c_wchar_p
+LPDWORD = PDWORD = ctypes.POINTER(DWORD)
 
 INFINITE = 0xffffffff
 EXCEPTION_MAXIMUM_PARAMETERS = 15

--- a/vtrace/platforms/win32.py
+++ b/vtrace/platforms/win32.py
@@ -29,7 +29,7 @@ import envi.archs.amd64 as e_amd64
 import envi.symstore.resolver as e_resolv
 import envi.symstore.symcache as e_symcache
 
-from ctypes import wintypes
+import ctypes
 from ctypes import *
 
 logger = logging.getLogger(__name__)
@@ -46,15 +46,16 @@ IsWow64Process = None
 # Setup some ctypes helpers:
 # NOTE: we don't use LPVOID because it can return None.
 #       c_size_t is the designated platform word width int.
-LPVOID = c_size_t
-HANDLE = LPVOID
-SIZE_T = LPVOID
-QWORD  = c_ulonglong
-DWORD  = c_ulong
-WORD   = c_ushort
-BOOL   = c_ulong
-BYTE   = c_ubyte
-NULL   = 0
+LPVOID  = c_size_t
+HANDLE  = LPVOID
+SIZE_T  = LPVOID
+QWORD   = c_ulonglong
+DWORD   = c_ulong
+WORD    = c_ushort
+BOOL    = c_ulong
+BYTE    = c_ubyte
+NULL    = 0
+LPCWSTR = LPWSTR = c_wchar_p
 
 INFINITE = 0xffffffff
 EXCEPTION_MAXIMUM_PARAMETERS = 15
@@ -901,22 +902,22 @@ if sys.platform == "win32":
     kernel32 = windll.kernel32
     # We need to inform some of the APIs about their args
     kernel32.GetLastError.argtypes = []
-    kernel32.GetLastError.restype = wintypes.DWORD
+    kernel32.GetLastError.restype = DWORD
     kernel32.OpenProcess.argtypes = [DWORD, BOOL, DWORD]
     kernel32.OpenProcess.restype = HANDLE
     kernel32.CreateProcessA.argtypes = [LPVOID, c_char_p, LPVOID, LPVOID, c_uint, DWORD, LPVOID, LPVOID, LPVOID, LPVOID]
     kernel32.ReadProcessMemory.argtypes = [HANDLE, LPVOID, LPVOID, SIZE_T, LPVOID]
-    kernel32.ReadProcessMemory.restype = wintypes.BOOL
+    kernel32.ReadProcessMemory.restype = BOOL
     kernel32.WriteProcessMemory.argtypes = [HANDLE, LPVOID, c_char_p, SIZE_T, LPVOID]
-    kernel32.WriteProcessMemory.restype = wintypes.BOOL
+    kernel32.WriteProcessMemory.restype = BOOL
     kernel32.GetThreadContext.argtypes = [HANDLE, LPVOID]
     kernel32.SetThreadContext.argtypes = [HANDLE, LPVOID]
     kernel32.CreateRemoteThread.argtypes = [HANDLE, LPVOID, SIZE_T, LPVOID, LPVOID, DWORD, LPVOID]
     kernel32.SuspendThread.argtypes = [HANDLE,]
     kernel32.ResumeThread.argtypes = [HANDLE,]
-    kernel32.ResumeThread.restype = wintypes.DWORD
-    kernel32.ContinueDebugEvent.argtypes = [wintypes.DWORD, wintypes.DWORD, wintypes.DWORD]
-    kernel32.ContinueDebugEvent.restype = wintypes.BOOL
+    kernel32.ResumeThread.restype = DWORD
+    kernel32.ContinueDebugEvent.argtypes = [DWORD, DWORD, DWORD]
+    kernel32.ContinueDebugEvent.restype = BOOL
     kernel32.VirtualQueryEx.argtypes = [HANDLE, LPVOID, LPVOID, SIZE_T]
     kernel32.DebugBreakProcess.argtypes = [HANDLE,]
     kernel32.CloseHandle.argtypes = [HANDLE,]
@@ -931,15 +932,15 @@ if sys.platform == "win32":
     kernel32.FormatMessageW.argtypes = [DWORD, LPVOID, DWORD, DWORD, LPVOID, DWORD, LPVOID]
     kernel32.WaitForDebugEvent.argtypes = [LPVOID, DWORD]
 
-    kernel32.QueryDosDeviceW.argtypes = [wintypes.LPCWSTR, wintypes.LPWSTR, wintypes.DWORD]
-    kernel32.FindFirstVolumeW.argtypes = [wintypes.LPWSTR, wintypes.DWORD]
-    kernel32.FindFirstVolumeW.restype = wintypes.HANDLE
-    kernel32.FindNextVolumeW.argtypes = [wintypes.HANDLE, wintypes.LPWSTR, wintypes.DWORD]
-    kernel32.FindNextVolumeW.restype  = wintypes.BOOL
+    kernel32.QueryDosDeviceW.argtypes = [LPCWSTR, LPWSTR, DWORD]
+    kernel32.FindFirstVolumeW.argtypes = [LPWSTR, DWORD]
+    kernel32.FindFirstVolumeW.restype = HANDLE
+    kernel32.FindNextVolumeW.argtypes = [HANDLE, LPWSTR, DWORD]
+    kernel32.FindNextVolumeW.restype  = BOOL
     kernel32.FindVolumeClose.argtypes = [HANDLE]
-    kernel32.FindVolumeClose.restype = wintypes.BOOL
-    kernel32.GetVolumePathNamesForVolumeNameW.argtypes = [wintypes.LPWSTR, wintypes.LPWSTR, wintypes.DWORD, wintypes.PDWORD]
-    kernel32.GetVolumePathNamesForVolumeNameW.restype = wintypes.BOOL
+    kernel32.FindVolumeClose.restype = BOOL
+    kernel32.GetVolumePathNamesForVolumeNameW.argtypes = [LPWSTR, LPWSTR, DWORD, PDWORD]
+    kernel32.GetVolumePathNamesForVolumeNameW.restype = BOOL
 
     IsWow64Process = getattr(kernel32, 'IsWow64Process', None)
     if IsWow64Process is not None:
@@ -1367,13 +1368,13 @@ class WindowsMixin:
             size = kernel32.QueryDosDeviceW(namevalu, devicename, 1024)
 
             drivenames = create_unicode_buffer(1024)
-            valulen = wintypes.DWORD()
+            valulen = DWORD()
 
             stat = kernel32.GetVolumePathNamesForVolumeNameW(buffer.value, drivenames, 1024, pointer(valulen))
             if drivenames.value:
                 self.dosdevs.append((drivenames.value, devicename.value))
 
-            nextvolu = kernel32.FindNextVolumeW(hndl, buffer, wintypes.DWORD(1024))
+            nextvolu = kernel32.FindNextVolumeW(hndl, buffer, DWORD(1024))
             if not nextvolu:
                 err = kernel32.GetLastError()
                 if err == ERROR_NO_MORE_FILES:

